PathKit/Sources/PathKit/Vector.swift

```diff
@@ -50,12 +50,21 @@ extension Vector {
         self.dy = Double(dy)
     }
 
-    /// Creates a vector pointing to `point`.
+    /// Creates a vector pointing from the origin to `point`.
     ///
     /// - Parameter point: The point used to create the vector.
     public init(_ point: Point) {
-        dx = point.x
-        dy = point.y
+        self.init(tail: .zero, head: point)
+    }
+
+    /// Creates a vector pointing from `tail` to `head`.
+    ///
+    /// - Parameters:
+    ///   - tail: The point representing the tail of the vector.
+    ///   - head: The point representing the head of the vector.
+    public init(tail: Point, head: Point) {
```

Comment:
Why is tail first?

Reply:
![image](https://user-images.githubusercontent.com/2954988/51929529-4e363300-23ad-11e9-84f3-7ee67c403689.png)

Reply:
That was my reasoning

Reply:
Whats the reasoning for putting head first?

Reply:
ok, makes sense